qa/tasks/mgr/dashboard/test_cephfs.py

```diff
@@ -54,24 +52,24 @@ def _ls_dir(self, path, expectedLength, depth, baseApiPath):
         self.assertEqual(len(data), expectedLength)
         return data
 
-    def setQuotas(self, bytes=None, files=None):
+    def set_quotas(self, max_bytes=None, files=None):
```

Comment:
renamed to max_bytes because bytes is a python keyword

Reply:
Nevertheless you should rename `files` than to `max_files` as well to have the same level of description for both parameters.